packages/arb-token-bridge-ui/src/hooks/TransferPanel/useStyles.ts

```diff
@@ -0,0 +1,92 @@
+import { useMemo } from 'react'
+import { Chain } from 'wagmi'
+
+import { isNetwork } from '../../util/networks'
+import { GasEstimationStatus } from '../../components/TransferPanel/TransferPanelSummary'
+
+export function useStyles({
+  l1Network,
+  l2Network,
+  isSwitchingL2Chain,
+  isTransferring,
+  isDepositMode,
+  disableDeposit,
+  disableWithdrawal,
+  gasEstimationStatus
+}: {
+  l1Network: Chain
+  l2Network: Chain
+  isSwitchingL2Chain: boolean
+  isTransferring: boolean
+  isDepositMode: boolean
+  disableDeposit: boolean
+  disableWithdrawal: boolean
+  gasEstimationStatus: GasEstimationStatus
+}) {
+  const depositButtonColorClassName = useMemo(() => {
+    const { isArbitrum, isArbitrumNova, isXaiTestnet, isStylusTestnet } =
+      isNetwork(l2Network.id)
+
+    if (isArbitrumNova) {
+      return 'bg-arb-nova-dark'
+    }
+
+    if (isArbitrum) {
+      return 'bg-arb-one-dark'
+    }
+
+    if (isXaiTestnet) {
```

Comment:
Not introduced by this PR, but maybe we should put `isXaiTestnet` and `isStylusTestnet` above `isArbitrumNova`. Xai and Stylus are technically Arbitrum chains, and even if the code returns false for that we should cover the precise networks first, so if we changed things up in the future `isArbitrum` won't give us false positive.

Might be worth to also check for `isOrbitChain` from `isNetwork` and return `'bg-orbit-dark'`, and have Arbitrum as default